### ISSUE TITLE
Try spatialized dynamic loss

### ISSUE THREAD

**Issue by jder** [m2lines/Samudra#589]
One possible explanation for the horizontal banding artifact is that thetao is generally easy to predict (slow-moving) except for small high-energy regions. So we end up scaling the loss for those channels quite high which then causes them to over-fit in the high-energy regions. Let's instead try spatializing the loss. 

First experiment went badly: [train](https://wandb.ai/ocean_emulators/default/runs/q97xwk7x), [eval](https://wandb.ai/ocean_emulators/default/runs/f61zywa0). Losses are generally higher across the board. 

Things to try:
* [ ] Re-weight loss scales so we still end up with the same overall mean loss scale (ie 1) to prevent overall drift in effective learning rate
* [ ] Others?

cc @adam-subel
